config/object_labels/recipe_oatmeal.yaml

```diff
@@ -46,6 +46,8 @@ labels:
     label: "measuring cup"
   - id: 25
     label: "container" # prep bowl
+  - id: 32
```

Comment:
Why are these ID values jumping so high?

Reply:
We're trying to keep the id values unique across all recipes, so ids 26-31 are used somewhere else